pkg/yurthub/server/server.go

```diff
@@ -63,6 +63,11 @@ func NewYurtHubServer(cfg *config.YurtHubConfiguration,
 	rest *rest.RestConfigManager) (Server, error) {
 	hubMux := mux.NewRouter()
 	registerHandlers(hubMux, cfg, certificateMgr)
+	restCfg := rest.GetRestConfig(false)
+	clientSet, err := kubernetes.NewForConfig(restCfg)
+	if err != nil {
+		klog.Warningf("cannot create the client set: %v", clientSet)
```

Comment:
if the kube-client can not created, it's more reasonable to return here, not just log warning.

Reply:
If the network is disconnected, the program will be continue to execute and query the cache for the non-resource info. And I think the error should not be return.

Reply:
I think `kubernetes.NewForConfig` is used for creating client only, and do not access the remote kube-apiserver.

Reply:
I think you are right! And I will return the error.